x/executionlayer/client/util/util.go

```diff
@@ -64,3 +65,46 @@ func ProtobufSafeDecodeingHexString(str string) ([]byte, error) {
 
 	return res, nil
 }
+
+// GetLocalWalletInfo takes wallet info from local
+// Rules:
+// 1) If --from exists, search according to:
+//  (1) Local wallet alias
+//  (2) Address
+//  (3) Nickname
+// 2) If --from doesn't exist,
+//  (1) If only one wallet exists, take this
+//  (2) If not, raise error
+func GetLocalWalletInfo(valueFromFromFlag string, kb keys.Keybase, cdc *codec.Codec, cliCtx context.CLIContext) (keys.Info, error) {
+	if valueFromFromFlag != "" {
+		// Find in local wallet
+		key, err := kb.Get(valueFromFromFlag)
+		if err == nil {
+			return key, nil
+		}
+
+		// If not exist, try parsing into address and find in nickname
+		addr, err := GetAddress(cdc, cliCtx, valueFromFromFlag)
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse into address, or no registered address of the given nickname '%s'", valueFromFromFlag)
+		}
+		key, err = kb.GetByAddress(addr)
+		if err != nil {
+			return nil, err
+		}
+		return key, nil
+	}
+
```

Comment:
I think getbalnace should be possible without keys.
In the above case, it is impossible to search for an address that does not have a key.

```$ clif hdac getbalance --from friday1lgharzgds89lpshr7q8kcmd2esnxkfpwmfgk32```

Reply:
Damm, I did it too automatically

https://github.com/hdac-io/friday/pull/77/commits/b790e50f48a5acb0ade2d10970fa947ce67755ee

Reply:
Input key searching logic added if the input is not valid address nor registered nickname.

https://github.com/hdac-io/friday/pull/77/commits/96c8383242f9942b84422521489c5d8f522af74f

[Test log](http://132.145.81.228:8080/job/05_(Do%20not%20trigger%20yourself)%20Test%20run/37/console)